docs/redoc/master/openapi.json

```diff
@@ -230,6 +230,84 @@
         }
       }
     },
+    "/healthz": {
```

Comment:
It is indeed deprecated. But I decided to add it in case some tooling still depends on it.

Are you suggesting to keep the endpoint, but to only remove it from the OpenAPI spec?

Reply:
I was suggesting to drop it completely.

I guess if we have it, we should support it fully.

Fine by me to keep it as it is because it is an explicit choice you've made already :+1: